### PR TITLE
open-cluster-management/backlog#1573 -- more app work for conceptual …

### PR DESCRIPTION
**Pull request by bcpswope**
…area

### CODE REVIEW

File: manage_applications/deploying.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+For more information, see [Scheduling resource deployments for a subscription](managing_subscriptions.md#subscription_timewindow). -->
+
+## Promoting a deployable to a channel
+
+To promote a deployable to a channel, you can use any of the following methods:
+
+* Point the deployable to a specific channel by configuring the `spec.channels` field within the deployable definition with the correct annotations to identify the channel.
+
+    The `spec.channels` parameter is available for deployable (`deployable.apps.open-cluster-management.io`) resources to identify the channels where the deployable is to be promoted. The following example shows a deployable that defines the channels where the deployable is to be included.
```
Comment (cadawson) [inline manage_applications/deploying.md:33]: Consider moving the parentheses after the word resources.

File: manage_applications/deploying.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        overrides:
+        dependencies:
+        channels:
+        - mychannel
+        - otherchannel
+    ```
+
+* Update the channel definition to identify the deployables to include in the channel. Use the `spec.package` and `spec.packageFilter` fields to specify the deployables.
+
+  As an example, the following channel automatically pull the most recent, or latest, `nginx` chart when a new chart is published to the source Helm repository for the channel. The chart deployable must have a matching version `1.x` to be promoted to the channel.
```
Comment (cadawson) [inline manage_applications/deploying.md]: pulls...change made

File: manage_applications/deploying.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+* Update the channel definition to specify channel gate requirements and update the definitions for your deployables to include the fields and values to match the gate requirements.
+
+  Channel gate requirements are defined within the `spec.gate` section of a channel definition. If the deployable has the fields to match the channel `spec.gate` values, the deployable is promoted to the channel. In this case, the deployable does not need to point to a specific channel with the  `spec.channels` field.
+
+### Deploying with a percentage roll out  
+
+If you want to roll out a deployment to your target managed clusters instead of deploying to all target cluster, you can configure the deployment of a deployable or chart to only a percentage of your managed clusters at a time. For instance, you might want to roll out a deployment when you need to deploy an update but you do not want to affect all clusters at once. When the deployment is successful on a cluster, the deployment is rolled out to another cluster.
```
Comment (cadawson) [inline manage_applications/deploying.md]: all target clusters...change made

File: manage_applications/deployment_app.md [hunk 4]
```diff
@@ -1,144 +1,14 @@
...
-
-* Placement rules (`PlacementRule.app.ibm.com`) define the target clusters where deployables can be deployed. Use placement rules to help you facilitate the multi-cluster deployment of your deployables.
+Placement rules (`placementrule.apps.open-cluster-management.io`) define the target clusters where subscriptions deploy and maintain the Kubernetes resources. You can use placement rules to help you facilitate the multi-cluster deployment. Placement rules that are defined as stand-alone resources can be shared across deployables.
 
 Channels, subscriptions, and placement rules can help you to unify and simplify your continuous deliveries that involve frequent updates and deployments to multiple clusters. Use a combination of these resources when you need to make deploy to multiple clusters, deploy frequent updates, and share deployment settings across multiple deployables.
```
Comment (cadawson) [inline manage_applications/deployment_app.md]: remove make...change made

File: manage_applications/deploying.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+2. If your deployables do not exist, create your deployables. For more information, see [Creating and managing deployables](managing_deployables.md).
+
+3. If your target cluster or clusters are not subscribed to the channel, create the subscription. For more information, see [Creating and managing subscriptions](managing_subscriptions.md).
+
+4. Define the placement rule for the deployables that are to be deployed from the channel. For more information, see [Creating and managing placement rules](managing_placement_rules.md).
+
+    A placement rule can be defined for a subscription and for deployables. Define the placement rule or reference to the placement rule in the subscription definition to have the rule apply to multiple deployables and clusters. Define the placement rule or reference to the rule in the deployables definitions when you do not want to have the same settings apply to all deployables for the subscription or when you want to override a subscription-level placement rule. If you are referencing a placement rule, you can also include override settings for a deployable to override some placement rule values with values specific to the deployable.
```
Comment (dockerymick) [inline manage_applications/deploying.md:13]: This paragraph seems like a concept. I would update the step to say: "Define the placement rule for the deployables that are to be deployed from the channel. A placement rule can be defined for a subscription and for deployables. For more information, see ...

* Define the placement rule or reference to the placement rule in the subscription definition to have the rule apply to multiple deployables and clusters.

* Define the placement rule or reference to the rule in the deployables definitions to have unique settings applied to all deployables for the subscription, or when you want to override a subscription-level placement rule."

File: manage_applications/deploying.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+
+5. Optional. If you created the placement rule as a stand-alone resource, edit the definition for your subscription or deployables to reference your placement rule.
+
+6. Edit the definition for your deployables and your channel to ensure that deployables are promoted to the channel. For more information, see [Promoting a deployable to a channel](#promoting-a-deployable-to-a-channel).
```
Comment (dockerymick) [inline manage_applications/deploying.md:17]: I am not sure that this anchor will build in Pantheon. To workaround anchors, I have explicitly stated that a section is within the page. For example, I would suggest: "For more information, see the following section on the page, _Promoting a deployable to a channel_."

File: manage_applications/deploying.md (hunk 1 above)
Comment (bcpswope) [reply manage_applications/deploying.md:33]: not in this context, these are the apis ;-)

File: manage_applications/deploying.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+
+If you need to deploy new or changed Helm charts or other resources during only specific times, you can define subscriptions for those resources to begin deployments during only those specific times. Alternatively, you can restrict deployments from beginning during specific time windows, such as to avoid unexpected deployments during peak business hours.
+
+For more information, see [Scheduling resource deployments for a subscription](managing_subscriptions.md#subscription_timewindow). -->
+
+## Promoting a deployable to a channel
+
+To promote a deployable to a channel, you can use any of the following methods:
+
+* Point the deployable to a specific channel by configuring the `spec.channels` field within the deployable definition with the correct annotations to identify the channel.
```
Comment (dockerymick) [inline manage_applications/deploying.md:31]: Reference a specific channel in the deployable by configuring the `spec.channels` field with the correct annotations.

File: manage_applications/deploying.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+
+    The `spec.channels` parameter is available for deployable (`deployable.apps.open-cluster-management.io`) resources to identify the channels where the deployable is to be promoted. The following example shows a deployable that defines the channels where the deployable is to be included.
+
+    ```yaml
+    apiVersion: apps.open-cluster-management.io/v1
+    kind: Deployable
+    metadata:
+        name: deployable1
+        namespace: default
```
Comment (dockerymick) [inline manage_applications/deploying.md:40]: i think you should align `name` and `namespace` with the 't' in metadata...if that makes sense. Or validate the indentation is correct.

File: manage_applications/deploying.md [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+        placementRef:
+          kind: PlacementRule
+          name: towhichcluster
+    ```
+
+* Update the subscription definition to identify the deployables. The configuration for promoting a deployable to a channel can also be specified within the subscription definition.
+
+  In the previous example, `packageFilter.version: "1.36.x"` indicates the specific `nginx` version `1.36.x` chart is  promoted through the channel for deployment with the subscription.
+
+* Update the channel definition to specify channel gate requirements and update the definitions for your deployables to include the fields and values to match the gate requirements.
```
Comment (dockerymick) [inline manage_applications/deploying.md:89]: Update the channel definition to specify channel gate requirements, and update the definitions for your deployables to include the fields and values to match the gate requirements.

File: manage_applications/deploying.md [hunk 10]
```diff
@@ -0,0 +1,109 @@
+## Deploying by using channels, subscriptions, and placement rules
+
+To set up and use channels, subscriptions, and placement rules for deployments, complete the following procedure:
+
+1. If the channel for representing the object store, Kubernetes namespace, or Helm repository does not exist, create the channel. Ensure that you define any labels or annotations that deployables must have before they can be promoted to the channel. For more information, see [Creating and managing channels](managing_channels.md).
```
[the remark this answers is not in the document]
Comment (bcpswope) [reply manage_applications/deploying.md:5]: good feedback, will handle in another PR!

File: manage_applications/deploying.md (hunk 8 above)
Comment (bcpswope) [reply manage_applications/deploying.md:40]: I will check, but these are likely getting deleted.... (repeated and then finally removed)

File: manage_applications/deploying.md (hunk 9 above)
Comment (bcpswope) [reply manage_applications/deploying.md:89]: @dockerymick good stuff, will work in a new PR I have to create anyway!

**Pull request merged**